pkgs/by-name/gr/grayjay/package.nix

```diff
@@ -114,7 +114,16 @@ buildDotnetModule (finalAttrs: {
 
   nugetDeps = ./deps.json;
 
-  dotnet-sdk = dotnetCorePackages.sdk_9_0;
+  dotnet-sdk = dotnetCorePackages.sdk_9_0 // {
```

Comment:
@samfundev this removes the nuget dependencies that are tied to specific runtime versions, so the package won't break when we update dotnet.